### ISSUE TITLE
paths and moduleDirectory Conflict?

### ISSUE THREAD

**Issue by malamoney** [benmosher/eslint-plugin-import#1466]
I have the following in my .eslintrc.json file:

```
"settings": {
  "import/resolver": {
    "node": {
      "moduleDirectory": ["src"]
    }
  }
}
```

When the "moduleDirectory" property is set VS Code does not complain about absolute imports (import/no-unresolved)

However, when I run eslint from the command-line, it throws a bunch of import/no-unresolved errors. If I set the "paths" property instead of "moduleDirectory", eslint no longer complains when run from the command line.

```
"settings": {
  "import/resolver": {
    "node": {
      "paths": ["src"]
    }
  }
}
```

But then VS Code throws the import/no-unresolved errors. My natural thought was that I just need to set both "paths" and "moduleDirectory", but this does not seem to work. VS Code is fine, but eslint from command line is not. When "moduleDirectory" is present it seems that the "paths" property is ignored.

```
"settings": {
  "import/resolver": {
    "node": {
      "moduleDirectory": ["src"],
      "paths": ["src"]
    }
  }
}
```

Setting `"import/no-unresolved": "off"` gets rid of the errors, but that's not how I want to them to go away.

Thoughts?

**Comment by ljharb** [comment]
This is likely caused by your vscode settings not correctly scoping to the project dir.

The correct settings are always "what works on the command line"; anything else is a bug in yhour editor.
